frappe/desk/reportview.py

```diff
@@ -53,8 +53,7 @@ def get_count():
 		controller = get_controller(args.doctype)
 		data = controller.get_count(args)
 	else:
-		distinct = "distinct " if args.distinct == "true" else ""
-		args.fields = [f"count({distinct}`tab{args.doctype}`.name) as total_count"]
```

Comment:
```diff
-  count(distinct `tabTable`.name) as total_count
+  distinct count(`tabTable`.name) as total_count
```



![image](https://user-images.githubusercontent.com/9079960/216619574-2f240ecc-a2db-4d6d-9ace-e793c695ab54.png)


Pretty sure these two are not equivalent :sweat_smile: 

When joined the count will be ~# child table rows instead of # of parents which will give incorrect count on list views.

Reply:
Oopsie